qna/DE-fiskaltrust-sandbox.md

```diff
@@ -11,14 +11,12 @@ How can I test my implementation?
 What is the fiskaltrust sandbox?
 
 ## Metadata tags
-lang-en, market-all, middleware, portal
+lang-en, market-de, middleware
 
 ## Answer
 
 We offer a test system called sandbox. The sandbox is fully functional just like our live system. Depending on the country, it can be reached via the following URLs:
```

Comment:
If we are making it specific to Germany this line should be changed to: "For German market, the sandbox can be reached via the following URL"

Reply:
changed